packages/components/src/compare-filter/index.js

```diff
@@ -91,7 +91,7 @@ export class CompareFilter extends Component {
 	}
 
 	render() {
-		const { labels, type } = this.props;
+		const { labels, type, autocompleter } = this.props;
```

Comment:
What do you think about changing this to `searchAutocompleter`? That could make it more obvious that it is used by search.

Reply:
But then it would be inconsistent with `type` which is also kind of `searchType`:
```js
	/**
	 * Which type of autocompleter should be used in the Search
	 */
	type: PropTypes.string.isRequired,
	/**
	 * The custom autocompleter to be forwarded to the `Search` component.
	 */
	autocompleter: PropTypes.object,
```

and changing `type` would be a breaking change.

As mentioned [above](https://github.com/woocommerce/woocommerce-admin/pull/6911#issue-626319409) I was thinking of doing it another way around and provide all props together like:
```js
	/**
	 * Object of localized labels.
	 */
	searchProps: PropTypes.shape( {
		placeholder: PropTypes.string,
		type: PropTypes.string.isRequired,
		autocompleter: PropTypes.object,
	} ),
```
Or even accepting the entire component as a prop.

Personally, to keep the API and prop names consistent, I'd either:
1. Leave it as is, or 
2. Group all search props together.

WDYT?